demos/monge_ampere_3d.py

```diff
@@ -0,0 +1,94 @@
+# Monge-Ampère mesh movement in three dimensions
+# ====================================================================
+
+# In this demo we demonstrate that the  Monge-Ampère mesh movement
+# can also be applied to 3D meshes. We employ the `sinatan3` function
+# from :cite:`park2019` to introduce an interesting pattern.
+
+import movement
+from firedrake import *
+
+
+def sinatan3(mesh):
+    x, y, z = SpatialCoordinate(mesh)
+    return 0.1*sin(50*x*z)+atan2(0.1, sin(5*y)-2*x*z)
```

Comment:
Do you have the Black pre-commit hook installed? This doesn't seem to have been formatted.